packages/terra-card/src/CardPadding.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+  /**
+   * Provides 10px of themeable padding vertical
+   */
+  hasPaddingVertical: PropTypes.bool,
+  /**
+   * Provides 10px of themeable padding horizontal
+   */
+  hasPaddingHorizontal: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasPaddingHorizontal: false,
+  hasPaddingVertical: false,
+};
+
+
+const CardPadding = ({
```

Comment:
Instead of this being a sub-component of Card. This should be a global padding component that card either consumes or consumers can use. We don't want padding sub components added to multiple different terra components.